common.mk

```diff
@@ -301,6 +308,10 @@ filelist-tee-common: optee-client xtest helloworld
 			"$(HELLOWORLD_PATH)/ta/8aaaf200-2450-11e4-abe2-0002a5d5c51b.ta" \
 			"444 0 0" 					>> $(fl); \
 	fi
+	@if [ "$(QEMU_USERNET_ENABLE)" = "y" ]; then \
```

Comment:
Yeah I have a slight preference for doing this in `gen_rootfs` but I leave it up to you. You can keep this here, I don't mind that much because it does makes sense here.